lib/chef/knife/bootstrap_windows_base.rb

```diff
@@ -283,14 +283,6 @@ def bootstrap(proto=nil)
           warn_chef_config_secret_key
         end
 
-        bootstrap_architecture = Chef::Config[:knife][:bootstrap_architecture]
-        if bootstrap_architecture && ![:x86_64, :i386].include?(bootstrap_architecture.to_sym)
-          raise "Valid values for the knife config :bootstrap_architecture are i386 or x86_64. Supplied value is #{bootstrap_architecture}"
-        end
-        if Chef::Config[:knife][:architecture]
-          raise "Do not set :architecture in your knife config, use :bootstrap_architecture."
-        end
-
         validate_name_args!
```

Comment:
:+1: for what @mwrock said. We used to only provide 32-bit builds, which still work on all 64-bit Windows servers except Nano. There are some complications that would make someone stick with 32-bit until they were ready to deal with them, like file / registry redirection.

https://blog.chef.io/2015/11/19/pre-release-announcement-native-64-bit-windows-chef-client/